.github/workflows/build.yaml

```diff
@@ -184,6 +184,10 @@ jobs:
         with:
           version: ${{ env.KIND_VERSION }}
           image: ${{ env.KIND_IMAGE }}
+      - name: Test connection to Kubernetes cluster
+        run: |
+          kubectl cluster-info
+          kubectl describe node
```

Comment:
AFAIK this will only describe node even if it's is `NotReady` status. Ideally kind action should block and wait until nodes are ready, but what I've seen it's not the case.

Therefore, I believe we should `kubectl wait` for node before we run integration tests.

Reply:
Ah, learned something new. Thanks.

I added the requested wait. Do you still want to keep the `describe` for the information?

Reply:
🚀 Description might be useful one day.